### ISSUE TITLE
vbox build for `arch`

### ISSUE THREAD

**Issue by malice-0** [rylagek/pack_everything#23]
Repo contains template that needs to be tested on more computers.

A couple questions to answer:
1. what packages do we actually need? Is just `base` and `base-devel` sufficient?
2. Are there any barebones provisioners we should add? Maybe installing Docker?
3. Does the boot command timing work on other, less powerful machines?

**Comment by malice-0** [comment]
Also, the current setup is BIOS/GPT with boot, swap and root partitions, with the root partition using `ext4`. Is this desired or is there another configuration that makes more sense?

**Comment by malice-0** [comment]
@rylagek forgot to tag you

**Comment by rylagek** [comment]
That configuration is fine. We're doing a basic install for now so the exact partitions don't matter that much

**Comment by rylagek** [comment]
Running the build gave an error so it looks like we need to better utilize zsh:
`zsh: ./install.sh: bad interpreter: /bin/sh^M: no such file or directory`

**Comment by rylagek** [comment]
@atalkar did you get a zsh error?

**Comment by malice-0** [comment]
@rylagek No, `install.sh` is able to kick off for me, and artifacts are created

**Comment by rylagek** [comment]
Interesting. I'll look into what's going on with my build

**Comment by rylagek** [comment]
via `archiso` both `bash` and `sh` are located in `/usr/bin`
`install.sh` uses `#!/bin/sh` @atalkar does `which bash` or `which sh` return a different location for you?

**Comment by malice-0** [comment]
@rylagek `which` does return that. It seems like on the iso `/bin/sh` is symlinked to `/bin/bash`.

I tried changing the shebang to `/bin/zsh` for the scripts that run on the live image root (`install.sh` and `partition.sh`) and that seemed to work. I didn't look into it but it seems like zsh isn't in the packages we install, so `bootloader.sh` and `packer.sh` failed when I tried to use it, since they run with `/mnt` as the fs root.

**Comment by rylagek** [comment]
@atalkar interesting, I'll try to find some time to take a look at that this evening
